soc/arm/atmel_sam0/common/power.h

```diff
@@ -0,0 +1,35 @@
+/*
```

Comment:
This file is no longer necessary and can be removed. Functions declarations are available via `include/power.h`.

Reply:
removed it.

Reply:
oops, sorry - I hope I didn't miss anything else, but I think here I just used `rm` instead of `git rm` and forgot to commit it.